### DISCUSSION TITLE
爆照，许多人还不认识我吧

### DISCUSSION THREAD

**Discussion by 566001** [share121/inter-knot#1922] [其他]
![image](https://github.com/user-attachments/assets/4a670017-8146-4e61-92eb-80688b22b87f)

**Comment by akdjadpoaopsad** [comment]
bro有点小帅💀

**Comment by breeze0317** [comment]
目测楼主是新入学的学弟

**Comment by breeze0317** [comment]
（好好好，初中政治课本都来了是吧

**Comment by Tang-Li-1** [comment]
我去 威猛先生

**Comment by cpu644** [comment]
哥们可以去演元年港片

**Comment by bop114514** [comment]
我的妈呀，我是你的粉丝，主播你怎么来绳网了哈哈（

**Comment by Yitaruose-BaiXu** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >……
